tests/unit/type/hash-field-expire.tcl

```diff
@@ -0,0 +1,629 @@
+######## HEXPIRE family commands
```

Comment:
Github got confused and shows a huge diff for my commit here. Maybe looking it on git might help. It is better there.